hdi/ARM-templates/UBsetup/unravel-AWS-vm-4611-setup.sh

```diff
@@ -0,0 +1,122 @@
+echo "${DISKUUID}    /srv   ext4 defaults  0 0" >> /etc/fstab
+
+/usr/bin/mount -a
+
+# install unravel rpm
+rpmfile_name=`cat /tmp/rpmfilename`
+/usr/bin/rpm  -U $rpmfile_name
+
+/usr/bin/sleep 5
+
+
+# Update Unravel Lic Key into the unravel.properties file
+# Obtain a valid unravel Lic Key file ; the following is just non working one
+echo "com.unraveldata.lic=1p6ed4s492012j5rb242rq3x3w702z1l455g501z2z4o2o4lo675555u3h" >> /usr/local/unravel/etc/unravel.properties
+echo "export CDH_CPATH="/usr/local/unravel/dlib/hdp2.6.x/*"" >> /usr/local/unravel/etc/unravel.ext.sh
+
+# Update Azure blob storage account credential in unravel.properties file
+# Update and uncomment the following lines to reflect your Azure blob storage account name and keys
+
+if [ $BLOBSTORACCT != "NONE" ] && [ $BLOBPRIACKEY != "NONE" ] && [ $BLOBSECACKEY != "NONE" ]; then
+
+   echo "blob storage account name is ${BLOBSTORACCT}"
+   echo "blob primary access key is ${BLOBPRIACKEY}"
+   echo "blob secondary access key is ${BLOBSECACKEY}"
+   echo "# Adding Blob Storage Account information, Update and uncomment following lines" >> /usr/local/unravel/etc/unravel.properties
+   echo "com.unraveldata.hdinsight.storage-account-name-1=fs.azure.account.key.${BLOBSTORACCT}.blob.core.windows.net" >> /usr/local/unravel/etc/unravel.properties
+   echo "com.unraveldata.hdinsight.primary-access-key=${BLOBPRIACKEY}" >> /usr/local/unravel/etc/unravel.properties
+   echo "com.unraveldata.hdinsight.storage-account-name-2=fs.azure.account.key.${BLOBSTORACCT}.blob.core.windows.net" >> /usr/local/unravel/etc/unravel.properties
+   echo "com.unraveldata.hdinsight.secondary-access-key=${BLOBSECACKEY}" >> /usr/local/unravel/etc/unravel.properties
+
+else
+   echo "One or more of your blob storage account parameter is invalid, please check your parameter file"
+fi
+
+sleep 3
+
+if [ $DLKSTOREACCT != "NONE" ] && [ $DLKCLIENTAID != "NONE" ] && [ $DLKCLIENTKEY != "NONE" ] && [ $DLKCLITOKEPT != "NONE" ] && [ $DLKCLIROPATH != "NONE" ]; then
+
+   echo "Data Lake store name is ${DLKSTOREACCT}"
+   echo "Data Lake Client ID is ${DLKCLIENTAID}"
+   echo "Data Lake Client Key is ${DLKCLIENTKEY}"
+   echo "Data Lake Access Token is ${DLKCLITOKEPT}"
+   echo "Data Lake Client Root Path is ${DLKCLIROPATH}"
+   echo "# Adding Data Lake Account information, Update and uncomment following lines" >> /usr/local/unravel/etc/unravel.properties
+   echo "com.unraveldata.adl.accountFQDN=${DLKSTOREACCT}.azuredatalakestore.net" >> /usr/local/unravel/etc/unravel.properties
+   echo "com.unraveldata.adl.clientId=${DLKCLIENTAID}" >> /usr/local/unravel/etc/unravel.properties
+   echo "com.unraveldata.adl.clientKey=${DLKCLIENTKEY}" >> /usr/local/unravel/etc/unravel.properties
+   echo "com.unraveldata.adl.accessTokenEndpoint=${DLKCLITOKEPT}" >> /usr/local/unravel/etc/unravel.properties
+   echo "com.unraveldata.adl.clientRootPath=${DLKCLIROPATH}" >> /usr/local/unravel/etc/unravel.properties
+
```

Comment:
In addition to WASB and ADLS we also support ABFS (Azure datalake Gen 2)
`Step 5.C Properties for ADLS Gen 2 (ABFS) (Unravel 4.5.2.x)`
https://docs.unraveldata.com/en/azure-hdi-part1-vm.html#al_idp101525